cli/internal/context/testdata/duplicate-pkgs/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "repo-with-dupe-workspaces",
+  "workspaces": [
+    "apps/**"
```

Comment:
Just in case anybody models after this we want to avoid `**`
```suggestion
    "apps/*"
```

Reply:
updated. but why?